src/objects/zcl_abapgit_oo_base.clas.abap

```diff
@@ -12,21 +12,11 @@ CLASS zcl_abapgit_oo_base DEFINITION PUBLIC ABSTRACT.
   PRIVATE SECTION.
     DATA mv_skip_test_classes TYPE abap_bool.
 
-    METHODS deserialize_abap_source_old
-      IMPORTING is_clskey TYPE seoclskey
-                it_source TYPE zif_abapgit_definitions=>ty_string_tt
-      RAISING   zcx_abapgit_exception.
-
-    METHODS deserialize_abap_source_new
```

Comment:
this is deleted? but required in the code, is this some separate refactoring task? Its really difficult to review a fix with 1000+ lines changed with unrelated changes

Reply:
Not deleted: it's moved to the new class ZCL_ABAPGIT_OO_DESERIALIZER. These two methods are used only for interfaces (classes are deserialized in ZCL_ABAPGIT_OO_CLASS), so it should at least be in ZCL_ABAPGIT_OO_INTERFACE. I preferred to move everything to ZCL_ABAPGIT_OO_DESERIALIZER (same design as for the serialization, i.e. everything in class ZCL_ABAPGIT_OO_SERIALIZER). Let me rework that in 2 PRs.